package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfd-crm",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
I take it that this branch will be rebased with https://github.com/DEFRA/ffc-sfd-crm/pull/4 before final merge.

Reply:
Have now rebased with the above mentioned PR.